Makefile.in

```diff
@@ -134,7 +134,8 @@ ifeq ($(HAVE_APPARMOR),-DHAVE_APPARMOR)
 	sh -c "if [ ! -d $(DESTDIR)/$(sysconfdir)/apparmor.d ]; then install -d -m 755 $(DESTDIR)/$(sysconfdir)/apparmor.d; fi;"
 	install -c -m 0644 etc/firejail-default $(DESTDIR)/$(sysconfdir)/apparmor.d/.
 	sh -c "if [ ! -d $(DESTDIR)/$(sysconfdir)/apparmor.d/local ]; then install -d -m 755 $(DESTDIR)/$(sysconfdir)/apparmor.d/local; fi;"
-	install -c -m 0644 etc/firejail-local $(DESTDIR)/$(sysconfdir)/apparmor.d/local/.
```

Comment:
You forgot to revert this one. Perhaps all distros like Debian or [Arch](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=firejail-apparmor#n14) have mechanism to retain locally modified configs. Users really should avoid using `make install` outside package management.

Reply:
I agree, users should use the packages provided by the distribution.
But it looks like some are actually building from source and installing directly.
For those it's better to keep the check, as it would overwrite their local modifications otherwise.

Reply:
Hm, I somewhat misread that this conditional would break distros but it's just fine. Sorry for the noise.